docs/maintainers.md

```diff
@@ -125,6 +126,7 @@ The following people (used to) maintain DevDocs:
 - [Jed Fox](https://github.com/j-f1)
 - [Mrugesh Mohapatra](https://github.com/raisedadead)
 - [Oliver Eyton-Williams](https://github.com/ojeytonwilliams)
+- [Paul Sernatinger](https://github.com/oscillot)
```

Comment:
Hey @oscillot, if you'd like to become a maintainer, that's awesome. It's more of a long term thing, though, so if you want to stick around and continue finding ways to contribute, please do.

Basically, if you maintain DevDocs for a bit we can make it official!

Reply:
++ Understood, I can remove that for now